RossBlassingame/JPL-CUSeniorProjects
# Feature/documentation

What changes were made:
Comments and documentation

What are the relevant JIRA tickets:
94

How can these changes be tested(test #?):
n/a

Comments:
Please go through and make sure everything makes sense -- if there's something that should be said that I haven't, feel free to add it. If there's something that should be changed, feel free to change it. Additionally, once this pull request is completed and everything is in a state that everyone's happy with, this will be where the release will be created.
